tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -42,8 +42,8 @@ Make sure:
 
    c. The local checkout is at `main` branch: `git rebase-update`
 
-2. Your Flutter version is equal to latest candidate release branch:
-    - Run `devtools_tool update-flutter-sdk --from-path`
+2. Your Flutter version in `tool/flutter-sdk` is equal to latest candidate release branch:
+    - Run `devtools_tool update-flutter-sdk`
```

Comment:
this is a logic change that I'm not certain we want to make.

How about:

Your Flutter SDK in `tool/flutter-sdk` and the one on `PATH` are updated to the latest candidate release branch:
` Run `devtools_tool update-flutter-sdk --update-on-path`